babel.config.js

```diff
@@ -75,6 +75,7 @@ module.exports = (api) => {
 
             // we require polyfills for this already
             'Array.prototype.includes',
+            'Object.assign',
```

Comment:
we should still warn for this though, using Object.assign non-polyfilled is problematic (but we catch it with IE11 e2e tests I guess)